content/integrations/codepush-integration.md

```diff
@@ -0,0 +1,105 @@
+---
+title: CodePush integration
+description: How to integrate your workflows with CodePush using Codemagic
+weight: 22
+---
+
+[**CodePush**](https://github.com/microsoft/code-push) is a cloud service that enables React Native developers to deploy mobile app updates directly to their users’ devices without re-deploying them to the stores.
+
+## Step by step guide to configure CodePush
```

Comment:
How about saying sth like "If CodePush is already configured for your project, skip to step.." or maybe split the instructons to 2, project-side setup and Codemagic setup.